src/App.tsx

```diff
@@ -15,33 +48,89 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <div className="App">
-    <button type="button">
-      Start
-    </button>
-
-    <button type="button">
-      Sort alphabetically
-    </button>
-
-    <button type="button">
-      Sort by length
-    </button>
-
-    <button type="button">
-      Reverse
-    </button>
-
-    <button type="button">
-      Reset
-    </button>
-
-    <ul className="Goods">
-      <li className="Goods__item">Dumplings</li>
-      <li className="Goods__item">Carrot</li>
-      <li className="Goods__item">Eggs</li>
-      <li className="Goods__item">...</li>
-    </ul>
-  </div>
-);
+export const App: React.FC = () => {
+  const [isStarted, show] = useState(false);
```

Comment:
Function show should show. But your function can show and hide.

show(true) => action is show
show(false) => action is hide

If u want to have show function it should be:

```
const [isStarted, setIsStarted] = useState(false);

const start = () => setIsStarted(true); 
const finish (or end) = () => setIsStarted(false); 
```